### ISSUE TITLE
incorrect behavior when no viewbox set for a symbol tag

### ISSUE THREAD

**Issue by mgrudzinska** [Samsung/thorvg#1183]
in a case when no 'width/height/viewBox' values are set for the symbol node, the behavior is incorrect

**Comment by hermet** [comment]
@mgrudzinska please close this if this issue is resovable by https://github.com/Samsung/thorvg/pull/1184.

**Comment by mgrudzinska** [comment]
@hermet, sorry for such a poor issue description that isn't even in english... 
I'll try to sent a patch tmr, #1184 doesn't solve it

**Comment by mgrudzinska** [comment]
```
<svg viewBox="0 0 40 20" xmlns="http://www.w3.org/2000/svg">
  
  <symbol id="myDot" preserveAspectRatio="none" >
    <circle cx="1" cy="2" r="3"/>
  </symbol>

  <use  fill="red" href="#myDot" x="10" y="0"/>
  <use  width="14" height="2" href="#myDot" x="3" y="4"/>
</svg>
```

**Comment by mgrudzinska** [comment]
solved in #1200
